docs/glossary.md

```diff
@@ -0,0 +1,38 @@
+# AI Edge Terminology
+
+## Core
+
+- The central OpenShift cluster containing the tools responsible for creating any artifacts required for the successful deployment of an Inference Application to Near Edge environments.
+- There are no resources or network constraints expected in the core cluster as it is expected that it fully supports all workflows required for creating and verifying Inference Application container images.
+
+## Near Edge
+
+- This is a non-core distributed environment to run and serve AI/ML inference workloads in moderate yet constrained compute resources and network.
+- For the purpose of this repository, the near edge environment is represented by separate OpenShift clusters disconnected from the core, the internet or both but may be managed from a core OpenShift cluster.
+
+## Model Server
+
+- A Model Server is responsible for hosting models as a service "to return predictions based on data inputs that you provide through API calls."[^2]
+- For any workflows under opendatahub-io/ai-edge, we will be focusing on using the Model Servers and serving runtimes supported by Open Data Hub
+
+## Inference Application Container
+
+- OCI compliant container image[^3] with the models included during the build process
+- Support for container images where the model and model serving runtimes are stored together
+
+## Model Registry
+
+- A centralized repository for the models and their metadata and managing the model lifecycle and versions. Currently our pipelines only support S3 and Git as the source where models can be stored. 
+
+## OCI Distribution Registry
+
+- Open Container Initiative (OCI) compliant container registry where the model and other artifacts are stored and versioned ready to be deployed on production or staging environments.
```

Comment:
Here we assume that the model will be in the registry ... but above we say that we support models only in S3 or git repository.

So are we currently able to use container registries as sources of models?

Reply:
My thought was that quay is an OCI compliant registry and thats where our model containers are stored at the end of the pipeline, and hence this definition. We are only supporting s3 and git as the source of the models. Does that make sense?

Reply:
Right, but then we should say "where the Inference Application Container images are stored ...", not "where the model and other artifacts are stored ..." because the current wording suggests that we are ready to support something more being fetched from the image registry (for example models during the build pipeline) than we are.

Reply:
Makes sense, I can update the definition to : 

`Open Container Initiative (OCI) compliant container registry where the Inference Application Container images and other artifacts are stored and versioned ready to be deployed on production or staging environments.`